EIPS/eip-7488.md

```diff
@@ -0,0 +1,51 @@
+---
+eip: 7488
+title: Ground Zero
+description: Implements protocol and governance changes required by the community
+author: ProphetXZ (@ProphetXZ)
+discussions-to: https://ethereum-magicians.org/t/eip-ground-zero/15480
+status: Draft
+type: Standards Track
+category: Core
+created: 2023-06-30
+---
+
+## Abstract
+
+This EIP "Ground Zero" suggests to switch mainnet to permissioned to lower gas prices, transaction costs, and improve privacy. The adjustment requires protcol and governance changes. The EIP idea is braking and stopping the IPFS and Swarm store and broadcasting and instead suggests scalling down ethereum ressources, so that the community may be served in a more **sustainable way** in the future. In addition, to mitigate the currently existing problem with increasingly high gas prices, ethereum would benefit from switching to a permissioned mainnet (after successfully deploying a testnet version). This would greatly lower the number of requests and decrease the load for operating nodes. A further improvement would be to remove legacy APIs, so nodes no longer answer unnecessary public API requests. 
```

Comment:
- You should not reference your own EIP by your title. Just use "This EIP", without the title.
- This EIP doesn't suggest. Use "This EIP switches mainnet..."
- "The adjustment requires protcol and governance changes." is implied by this being `category: Core`. Remove it.
- Remove all mentions of IPFS/Swarm.
- "Braking" is ill-defined here. Use a different word, like "stopping."
- There's a typo in scaling.
- Actually, what you are describing doesn't really scale Ethereum up or down. Remove that part entirely.
- You mention sustainability here, but I don't see that in the Motivation. It's an argument for this EIP. Put that in the Motivation.
- Ethereum is capitalized.
- Testnets are an unrelated, irrelevant implementation detail. Remove that mention of them.
- Remove the last sentence mentionning "removing legacy APIs". It's unrelated to this EIP.